ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -113,53 +82,58 @@ message CreatedEvent {
   ContractMetadata metadata = 10;
 }
 
-// View of a create event matched by an interface filter.
-message InterfaceView {
+// Records that a contract has been transferred-out, and it becomes unusable on the source domain
```

Comment:
Just for my understanding: Are you actually planning to modify the existing ledger API in place, i.e., if we switch to that we will always get a new one or will there be a separate endpoint (either different port or different grpc service) for the multi-domain aware ledger API?

Reply:
No, this is a PR for communicating changes.
The actual to be merged PR will be placed in the canton-research/protobuf.
We plan to leave the v1 Ledger API as it is ATM. (later we still can decide to evolve v1, but that is a different conversation. Now we focus on getting the first iteration of the temporary PoC multi-domain Ledger API going, so CN integration work can start too.)